x-pack/plugins/security/server/config.ts

```diff
@@ -33,7 +33,11 @@ export const ConfigSchema = schema.object(
       schema.maybe(schema.string({ minLength: 32 })),
       schema.string({ minLength: 32, defaultValue: 'a'.repeat(32) })
     ),
-    sessionTimeout: schema.oneOf([schema.number(), schema.literal(null)], { defaultValue: null }),
+    sessionTimeout: schema.maybe(schema.oneOf([schema.number(), schema.literal(null)])), // DEPRECATED
```

Comment:
Had to leave this in otherwise the TypeScript static type checker blows a gasket.